### PR TITLE
feat(22393): implement jwks endpoint

### PR DESCRIPTION
**Pull request by nnorbert**
Implement jwks endpoint

### CODE REVIEW

**Review by peschina — APPROVED** [review]
LGTM

File: apps/journey-api/src/libraries/integratorEngine/plugins/formPlugin.ts [hunk 1]
```diff
@@ -50,8 +50,8 @@ export class FormPlugin implements IIntegratorPlugin {
     } as FormRunStepDataDO;
   }
 
-  //TODO: Add here the Forms Backend URL
   public getJwksUrl() {
-    return "/.well-known/jwks.json";
+    const url = new URL("/.well-known/jwks.json", process.env.FORMS_SERVICE_URL);
```
Comment (alfonsograziano) [inline apps/journey-api/src/libraries/integratorEngine/plugins/formPlugin.ts:54]: This endpoint is returning me 404. @nnorbert before merging is worth checking with Florian what the final endpoint will be

Comment (nnorbert) [reply apps/journey-api/src/libraries/integratorEngine/plugins/formPlugin.ts:54]: I have checked with Florian, he confirmed that the endpoint will be this, but the feature is not done yet.

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Pull request merged**